packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -122,6 +124,13 @@ class PerformanceMonitoring {
         (requestUrl.relative
           ? requestUrl.path
           : stripQueryStringFromUrl(requestUrl.href))
+
+      if (!transactionService.getCurrentTransaction()) {
+        transactionService.startTransaction(spanName, HTTP_REQUEST_TYPE, {
```

Comment:
Are we using the same spanName as transaction name for consistency or is there any reasoning behind this?